### ISSUE TITLE
Remove witDetail from model.noteLike

### ISSUE THREAD

**Issue by hcayless** [TEIC/TEI#1845]
Its participation in model.noteLike means `<witDetail>` shows up everywhere, like a fungus. It's really only useful in association with `<app>`. It makes most sense to put a `<witDetail>` next to its corresponding `<lem>` or `<rdg>`, though the GLs have examples with it following the `<app>`. Semantically, I agree that it's a sort of note, but that doesn't mean it should be able to occur everywhere a regular note can.

**Comment by sydb** [comment]
I agree wholeheartedly! Two concerns about actually doing this, though:
1) Deciding where it should be allowed (since there is even an example with it outside the `<app>`) may be a bit thorny. Is it reasonable to collect them all in a `<div type="witness_info">` inside `<back>`, e.g.?
2) Backwards incompatible change for which it is difficult (but not impossible) to give nice deprecation warnings

**Comment by hcayless** [comment]
Yeah, I think we should steer people toward putting it in `<app>`, but we could start by putting it in model.global.edit and adding it explicitly to `<app>`‘s content model. I’d be surprised if anyone’s collecting them in random weird spots.

**Comment by hcayless** [comment]
Yeah, I think we should steer people toward putting it in `<app>`, but we could start by putting it in model.global.edit and adding it explicitly to `<app>`‘s content model. I’d be surprised if anyone’s collecting them in random weird spots.

**Comment by hcayless** [comment]
@peterstadler: mind if I steal this from you? I was irritated all over again by seeing `<witDetail>` show up in James's new `<object>` stuff. WHERE IT ABSOLUTELY SHOULD NEVER OCCUR.

**Comment by martinascholger** [comment]
Seems that there is broad consent to this. Marking this ticket green.
